x11-misc/xfractint/xfractint-20.04_p15.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+MY_P=${P/_}
+
+DESCRIPTION="A fractal generator"
+HOMEPAGE="https://www.fractint.org"
+SRC_URI="https://www.fractint.org/ftp/current/linux/${MY_P}.tar.gz"
+
+LICENSE="free-noncomm HPND public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="x11-libs/libX11"
+
+DEPEND="${RDEPEND}
+	x11-libs/libXft"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+	"${FILESDIR}/xfractint-20.04p09-ldflags.patch"
+)
+
+src_compile() {
+	# Untested, any x86 archteam dev. is allowed to uncomment this.
+	local myasm="foo"
+	# use x86 && myasm="/usr/bin/nasm"
+	emake CC="$(tc-getCC)" AS="${myasm}" OPT="${CFLAGS}" LDFLAGS="${LDFLAGS}"
```

Comment:
This looks like some debug lines. You should probably remove them. You can use *toolchain-funcs.eclass* to get proper AS value, could be used for CFLAGS and LDFLAGS too.